mods/ts/rules/gdi-structures.yaml

```diff
@@ -262,9 +262,6 @@ GAWEAP:
 	WithIdleOverlay@TURBINES:
 		RequiresCondition: !build-incomplete
 		Sequence: idle-turbines
-	WithIdleOverlay@BIB:
-		RequiresCondition: !build-incomplete
-		Sequence: bib
```

Comment:
Is this an accident? It doesn't seem to get replaced with anything.

Reply:
The sequences were set up incorrectly. In the original game `gaweapbb.shp` is used with `gaweap.shp` when the building is idle, and both sprites are swapped out with `gaweap_1.shp` and `gaweap_2.shp` when the door opens.

We are using `gaweap_1.shp` and `gaweap_2.shp` when idle too, so the `gaweapbb.shp` was actually drawing a second redundant bib over the top of the normal bib.